src/libs/actions/OnyxUpdates.ts

```diff
@@ -42,6 +43,12 @@ function applyHTTPSOnyxUpdates(request: Request, response: Response) {
             if (response.jsonCode === 200 && request.successData) {
                 return updateHandler(request.successData);
             }
+
+            if (response.jsonCode === 408 && response.message?.includes('The account you are trying to use is deleted.')) {
+                signOutAndRedirectToSignIn(true);
+                return Promise.resolve();
+            }
```

Comment:
This does not belong to the onyx middleware. Let's write a new middleware for this flow.